src/main/java/teammates/logic/core/LoggingLogic.java

```diff
@@ -0,0 +1,81 @@
+package teammates.logic.core;
+
+import java.time.Instant;
+import java.util.List;
+
+import com.google.cloud.MonitoredResource;
+import com.google.cloud.logging.LogEntry;
+import com.google.cloud.logging.Payload.StringPayload;
+import com.google.cloud.logging.Severity;
+
+import teammates.common.util.Config;
+import teammates.common.util.Logger;
+import teammates.logic.core.LoggingService.LogSearcher;
+
+/**
+ * Handles the logic related to logging.
+ */
+public final class LoggingLogic {
+
+    private static LoggingLogic instance = new LoggingLogic();
+
+    private static final Logger log = Logger.getLogger();
+
+    private static final String LOG_NAME = "audit-logs";
+    private static final Severity LOG_LEVEL = Severity.INFO;
+    private static final String LOG_TYPE_ACCESS = "access";
+    private static final String LOG_TYPE_SUBMIT = "submit";
+
+    private LoggingLogic() {
+        // prevent initialization
+    }
+
+    public static LoggingLogic inst() {
+        return instance;
+    }
+
+    /**
+     * Creates a feedback session log.
+     */
+    public void createFeedbackSessionLog(String courseId, String email, boolean isAccess) {
+        if (Config.isDevServer()) {
+            // Not supported in dev server
+            return;
+        }
+        String payload = "Feedback session log: courseId=" + courseId + " email=" + email;
+        String type = isAccess ? LOG_TYPE_ACCESS : LOG_TYPE_SUBMIT;
+        LogEntry entry = LogEntry.newBuilder(StringPayload.of(payload))
```

Comment:
Actually do we need the payload since we have the labels already?

Reply:
I'm also wondering what the payload should be! We don't strictly need it, but it seems a bit strange to have logs with no description to them in the console. Since we shouldn't be exceeding the log storage limit, it might be helpful to have some form of payload